### DISCUSSION TITLE
Why is the module so big?

### DISCUSSION THREAD

**Discussion by silverwind** [vuejs/pinia#2128] [Help and Questions] [answered]
Evaluating this module and comparing it to similar module [`zustand`](https://github.com/pmndrs/zustand) for React, I can't help but notice that this  module is almost 7 times the size of `zustand`:

- https://bundlephobia.com/package/pinia@2.0.34
- https://bundlephobia.com/package/zustand@4.3.7

Why is it so big? Are there more lightweight alternatives?

**Comment by posva** [comment, marked as answer]
Because bundlephobia doesn't correctly work in this scenario and keeps all of the dev-only code which turns out to be way larger than the actual library code. In reality, pinia is around 1.5 kb min gzip
